tally/TrackLengthMeshTally.cpp

```diff
@@ -272,13 +276,31 @@ void TrackLengthMeshTally::write_data(double num_histories) {
         rel_err = sqrt((error / (tally * tally)) - (1. / num_histories));
       }
 
-      rval = mb->tag_set_data(error_tags[j], &t, 1, &rel_err);
-      assert(rval == MB_SUCCESS);
+      tally_vect[j] = score;
+      error_vect[j] = rel_err;
+
+      // TODO: It would be convenient for users to be able to get the total
```

Comment:
So right now the Tally API forbids one from accessing groups higher than N_GROUPS-1, we could easily change the API and then it would be a trivial change.